wgengine/monitor/monitor_windows.go

```diff
@@ -1,40 +1,22 @@
-// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
We don't bump years on edits. Only on new files. (same policy as Google open source, which is documented somewhere publicly but I always forget where otherwise I'd link it)

Reply:
Oh, yes, I know, sorry. I've started this file as a new one then renamed, thus the error...